Tasks/DockerComposeV0/utils.ts

```diff
@@ -1,10 +1,35 @@
 "use strict";
+import * as tl from "azure-pipelines-task-lib/task";
 import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import * as fileutils from "docker-common-v2/fileutils";
 
 export function getFinalComposeFileName(): string {
     return ".docker-compose." + Date.now() + ".yml"
 }
 
 export function writeFileSync(filename: string, data: any, options?: { encoding?: string; mode?: number; flag?: string; }): void {
     fs.writeFileSync(filename, data, options);
+}
+
+function getTaskOutputDir(command: string): string {
+    let tempDirectory = tl.getVariable('agent.tempDirectory') || os.tmpdir();
+    let taskOutputDir = path.join(tempDirectory, "task_outputs");
+    return taskOutputDir;
+}
+
+export function writeTaskOutput(commandName: string, output: string): string {
+    let taskOutputDir = getTaskOutputDir(commandName);
+    if (!fs.existsSync(taskOutputDir)) {
+        fs.mkdirSync(taskOutputDir);
+    }
+
+    let outputFileName = commandName + "_" + Date.now() + ".txt";
+    let taskOutputPath = path.join(taskOutputDir, outputFileName);
+    if (fileutils.writeFileSync(taskOutputPath, output) == 0) {
+        tl.warning(tl.loc('NoDataWrittenOnFile', taskOutputPath));
+    }
+    
```

Comment:
You could set the outputVariable value here

Reply:
The reason I'm not setting the output variable here is because there could be multiple files created.

For example if you'd execute the dockercomposedigest command it'll write one output image for every pull.

Do you want me to still change this? And if so, shall I then obtain the current value of the outputvariable there and apply it to it? (I don't really like this approach)

Reply:
Would you really want 2 different files though? You could append all logs in a single task execution to one file right?

Reply:
The main reason again is, is because the DockerV2 task also does this. So I assumed there should be a good reason for this.

The main reason for me would be is to have one log file per docker-compose command that's being executed. Which I think would make sense as we're not including logging from the task itself into the output variables.

Reply:
But given the naming convention to be generic, how would you figure out which docker-compose command created which file?   
If we have it in a single file, the number of file reads would reduce to 1. And, you could introduce a delimiter like `####` or something like that to differentiate the logs from different commands.